PowerEditor/src/WinControls/WindowsDlg/WindowsDlg.cpp

```diff
@@ -803,17 +803,20 @@ void WindowsDlg::resetSelection()
 {
 	assert(_pTab != nullptr);
 
-	auto curSel = _pTab->getCurrentTabIndex();
-	int pos = 0;
-	for (vector<int>::iterator itr = _idxMap.begin(), end = _idxMap.end(); itr != end; ++itr, ++pos)
+	if(_pTab)

```

Comment:
1. please add a white space between `if` and `(`
2. I would use : 
```cpp
    if (!_pTab)
        return;
```
and keep the rest unchanged.

3. `assert(_pTab != nullptr);` can be removed.

Reply:
removed assert and used immediate check for null with return as you requested. Adapted it to the same format with !_pTab also in doSort() method to be consistent within this file